test/functional/notifications_consultation_reminders_test.rb

```diff
@@ -3,15 +3,17 @@
 class NotificationsConsultationRemindersTest < ActionMailer::TestCase
   setup do
     @consultation = build(:consultation)
+    author = build(:author)
+    @consultation.update(authors: [author, author])
   end
 
   test "reminder emails should contain the title text and weeks remaining" do
-    @email = Notifications.consultation_deadline_upcoming(@consultation, weeks_left: 2)
-    assert_includes @email.body.to_s, %(Publish the government response for "#{@consultation.title}" within 2 weeks)
+    @email = MultiNotifications.consultation_deadline_upcoming(@consultation, weeks_left: 2)
+    assert_includes @email.first.body.to_s, %(Publish the government response for "#{@consultation.title}" within 2 weeks)
```

Comment:
Maybe also add an assert that checks only one email was sent - this would effectively check that one author doesn't receive two emails, what do you think?